dashboard/app/models/ability.rb

```diff
@@ -63,6 +64,7 @@ def initialize(user)
 
       if user.teacher?
         can :manage, Section, user_id: user.id
+        can :read, Section, user_id: user.id
```

Comment:
Since we're only asking for :read authentication later, I'm going to get rid of all references to :manage.